roles/elasticsearch.rb

```diff
@@ -1,3 +1,3 @@
 name "elasticsearch"
 description "Elasticsearch"
-run_list "recipe[consul]", "recipe[elasticsearch_wrapper_cookbook::elasticsearch]", "recipe[elasticsearch_wrapper_cookbook::consul_register]", "recipe[prometheus::elasticsearch_exporter]"
```

Comment:
have you tested this change? should we put consul_register before the elasticsearch or leave it as current changes?

Reply:
I don't think we should move consul_register before elasticsearch because it might trigger an alert when ES is not yet installed

Reply:
Make sense Luh, can you please check others roles? It seems the consul register installed before the service itself.

Reply:
It seems that we don't have standard for that. I see most roles run service before consul_register, the only roles that run consul_register first is barito-flow (both consumer and producer)